doc/local-pv-allocation.md

```diff
@@ -0,0 +1,128 @@
+# Local PV Allocation
+The purpose of this document is to show how to provision a VM disk on a specific node to be able to run it on the same node.
+
+### Prerequesites
+You have a Kubernetes cluster up and running with CDI installed.
+
+### Create your PersistentVolume manifest
+First create a PersistentVolume yaml file with NodeAffinity field to allocate the PV on specific node you choose. Verify 
+that the label you sepcify in the NodeAffinity exists on the node you want the PV to be allocated.
+In addition, add a specific label to be matched by a PersistenVolumeClaim during the binding process.
+An example of such PersistentVolume is:
+
+```yaml
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: local-pv-allocation
+  labels:
+    node: node02
+spec:
+  storageClassName: manual
+  persistentVolumeReclaimPolicy: Delete
+  capacity:
+    storage: 10Gi
+  local:
+    path: /mnt/local-storage/local/disk2
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - node02
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Filesystem
+
+```
+### Deploy the PersistentVolume to the cluster:
+```bash
+$ kubectl create -f manifests/example/local-pv-allocation.yaml
+```
+
+### Create your PersistenVolumeClaim manifest
+Create a PersistentVolumeClaim yaml file with a selector field that matches the lable you specified in the PersistentVolume.
+An example of such PersistentVolumeClaim is:
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: local-pv-allocation-claim
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
+  selector:
+    matchLabels:
+      node: node02
+  resources:
+    requests:
+      storage: 3Gi
+
+```
+
+### Deploy the PersistentVolumeClaim to the cluster:
+```bash
+$ kubectl create -f manifests/example/local-pv-allocation-claim.yaml
+```
+
+Verify that the PVC was deployed and is bound to your PV:
+```bash
+$ kubectl.sh get pvc local-pv-allocation-claim
+
+NAME                        STATUS    VOLUME                CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+local-pv-allocation-claim   Bound     local-pv-allocation   10Gi       RWO            manual         12m
+```
+
+### Update the PVC to start importing Disk Image
```

Comment:
Now that I think about it a little more, the rest is not need, since we are creating the PVC with the label selector, we should be able to specify the CDI annotations directly in the PVC request like usual, and we don't even need to update the PVC. The two step process was needed before when we were thinking of using a Pod to force the PVC to be allocated on a particular node, but this is better. So simply add the CDI annotation to import or whatever in the example PVC above.